### PR TITLE
Adding --timezone CLI Argument

### LINKED ISSUE #2961: Provide --timezone CLI argument
This is the most common reason people can't use CLI as a regular CLI and have to `java ... -jar presto-cli` instead.
Provide a direct support for this.

### PR DESCRIPTION
**Pull request by sanchitkashyap**
Fixes #2961

### CODE REVIEW

File: presto-cli/src/test/java/io/prestosql/cli/TestClientOptions.java [hunk 1]
```diff
@@ -134,6 +136,15 @@ public void testSessionProperties()
         assertEquals(new ClientSessionProperty("foo="), new ClientSessionProperty(Optional.empty(), "foo", ""));
     }
 
+    @Test
+    public void testTimeZone()
+    {
+        ClientOptions options = new ClientOptions();
+        options.timeZone = "UTC";
```
Comment (findepi) [inline presto-cli/src/test/java/io/prestosql/cli/TestClientOptions.java]: use a timezone that is never going to be tests' JVM zone
eg `Europe/Vilnius`

File: presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java [hunk 2]
```diff
@@ -152,6 +152,9 @@
     @Option(name = "--ignore-errors", title = "ignore errors", description = "Continue processing in batch mode when an error occurs (default is to exit immediately)")
     public boolean ignoreErrors;
 
+    @Option(name = "--timezone", title = "timezone", description = "TimeZone (default: system default time-zone)")
```
Comment (electrum) [inline presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java]: Let’s change description to

> Session time zone (default: system time zone)

File: presto-cli/src/test/java/io/prestosql/cli/TestClientOptions.java (hunk 1 above)
Comment (sanchitkashyap) [reply presto-cli/src/test/java/io/prestosql/cli/TestClientOptions.java]: Changed to Europe/Vilnius

File: presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java (hunk 2 above)
Comment (sanchitkashyap) [reply presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java]: Done

**Comment by findepi** [comment]
Merged, thanks!

**Pull request closed without merging**